app/src/main/java/edu/uc/reedws/musiclink/service/PlaylistService.kt

```diff
@@ -18,12 +18,20 @@ class PlaylistService(application: Application) {
         getLocalPlaylistDAO()
     }
 
+    /**
+     * Fetch the playlists added by user
+     *
+     */
     fun fetchPlaylists(): LiveData<List<PlaylistDTO>> {
         return localPlaylistDAO.getPlaylists()
     }
 
+    /**
+     * Create the playlist and save the playlist into room database
+     *
+     */
```

Comment:
Added an appropriate comment to indicate that this function creates and save new playlist inside room database